qcore/nhm.py

```diff
@@ -1,20 +1,49 @@
 from typing import List
+import datetime
+from dataclasses import dataclass
 
 import numpy as np
+from scipy.stats import truncnorm
 
+from srf_generation.source_parameter_generation.uncertainties import mag_scaling
 
+
+NHM_HEADER = f"""FAULT SOURCES - New Zealand National Seismic Hazard Model 2010 (created {datetime.datetime.now().strftime("%d-%b-%Y")}) 
+Row 1: FaultName 
+Row 2: TectonicType , FaultType 
+Row 3: LengthMean , LengthSigma (km) 
+Row 4: DipMean , DipSigma (deg) 
+Row 5: DipDir 
+Row 6: Rake (deg) 
+Row 7: RupDepthMean , RupDepthSigma (km) 
+Row 8: RupTopMean, RupTopMin RupTopMax  (km) 
+Row 9: SlipRateMean , SlipRateSigma (mm/yr) 
+Row 10: CouplingCoeff , CouplingCoeffSigma (mm/yr) 
+Row 11: MwMedian , RecurIntMedian  (yr) 
+Row 12: Num Locations on Fault Surface 
+Row 13+: Location Coordinates (Long, Lat) 
+"""
+
+# This mu is used to calculated Moment Rate
+MU = 3.0 * 10.0 ** 10.0
+
+
+def sample_trunc_norm_dist(mean, sigma, sigma_limit=2):
```

Comment:
This is a duplicate of the functions in srf_generation/source_parameter_generation/uncertainties/distributions.py
It's already here now. But could be imported from there with the same effect